src/requests/adapters.py

```diff
@@ -130,6 +130,7 @@ class HTTPAdapter(BaseAdapter):
         "_pool_connections",
         "_pool_maxsize",
         "_pool_block",
+        "urllib3_response_options",
```

Comment:
nit; the values in here could be anything for the `urlopen`/`_make_request` methods. Defining them as response_options seems too tightly scoped. I left another comment below, but perhaps `urllib3_request_kwargs` or just `urllib3_send_kwargs` would be more correct?

Reply:
I wanted to be very clear this week only affect one thing because naming things too generically makes others and they should work for other things. Based on your other comments, I'm thinking we should add the word `extra` to the band to be clear these are only extra args